components/make-static-previews.py

```diff
@@ -0,0 +1,145 @@
+import importlib
+import os
+import shutil
+import sys
+import time
+from pathlib import Path
+from typing import Generator
+
+import shiny.html_dependencies
+from _qmd import get_qmd_split
+from htmltools import Tag, head_content, tags
+
+compdir = Path(__file__).parent  # ./components
+rootdir = compdir.parent  # repository root
+cwd = Path(os.getcwd())  # current working directory
+libdir = compdir / "static" / "lib"
```

Comment:
Might be worth leaving a comment that this fixed directory collects dependencies from all examples in a shared folder.

Reply:
```suggestion
# This directory will be where we put all the dependencies from all the
# static component examples, rather than forcing the browser to fetch
# multiple redundant copies from various subfolders.
libdir = compdir / "static" / "lib"
```